modules/pool/lib/usePool.tsx

```diff
@@ -41,6 +41,31 @@ export function PoolProvider({ pool: poolFromProps, children }: { pool: GqlPoolU
     });
 
     const pool = (data?.pool || poolFromProps) as GqlPoolUnion;
+
+    // drop FTM as an invest option from the FreshBeets pool for now
+    let freshBeetsPool: GqlPoolUnion | null = null;
```

Comment:
can you create this as a useMemo var, don't want this to execute every single render